docs/api.rst

```diff
@@ -1,93 +1,25 @@
 API Reference
 =============
 
-
 ``aapns.api``
 -------------
 
+.. automodule:: aapns.api
+    :members:
 
-.. py:module:: aapns.api
-
-.. py:function:: create_client(client_cert_path, server, *, cafile=None)
-
-    This is a coroutine. Creates an APNS client.
-
-    :param str client_cert_path: Path to the client certificate to authenticate with
-    :param server: Server to connect to
-    :type server: :py:class:`aapns.config.Server`
-    :param srt cafile: Optional file path to server CA bundle
-    :return: An instance of :py:class:`APNS`
-    :rtype: :py:class:`APNS`
-
-.. py:class:: APNS
-
-    Main API of aapns. You should not instantiate it directly but rather use :py:func:`create_client` instead.
 
-    .. py:method:: send_notification(token, notification, *, apns_id=None, expiration=None, priority=Priority.normal, topic=None, collapse_id=None):
-
-        This is a coroutine.
-
-        Send a notification to the device registered for the given token. Returns
-        the notification ID.
-
-        :param str token: Device token
-        :param notification: The notification to send
-        :type notification: :py:class:`aapns.models.Notification`
-        :param str apns_id: Optional notification ID. If none is provided, APNS creates one
```

Comment:
How can this extra text be injected into auto module?

Reply:
I think I  could add comments to attributes to do this, but basically I've opted for "read the apple docs to understand what these do".